src/lib/pages/backoffice/Acquisition/Order/OrderSearch/OrderList/OrderListEntry.js

```diff
@@ -10,6 +10,7 @@ import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { Grid, Item, List } from 'semantic-ui-react';
+import get from 'lodash/get';
```

Comment:
Super minor, only of homogeneity:
```suggestion
import _get from 'lodash/get';
```